lib/opte/src/engine/tcp_state.rs

```diff
@@ -203,6 +221,13 @@ impl TcpFlowState {
                     return Some(CloseWait);
                 }
 
+                if tcp.has_flag(TcpFlags::SYN) {
```

Comment:
This is the remedy for the issue we saw in the colo?
- oxidecomputer/colo#61

Reply:
No, this is more related to #442 and its interaction with the `NewFlow` handling -- this is accounting for the fact that Established flows would allow non-FIN/RST packets through without issue. The consequence was that a new flow would appear considerably older / include packets from an older incarnation in its counts, so this is just for tracking statistics more correctly in such a case.

The remedy for that bug and/or #436  is the remainder of the patch: TIME_WAITs should be actually cleaned up, they won't filter packets, and a new SYN should practically always supersede flowtable state.